cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -719,6 +666,125 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
         }
     });
 
+    CustomSelfPacedGradingEditor = AbstractEditor.extend({
```

Comment:
So I'm not convinced this whole thing is necessary. Would it be possible to have the date editor just listen to the grading input (via the same event listener, just without being a combined class)? Or alternatively, having the grade editor letting the self-paced due date editor know that it should show once the right thing is selected? I'm not loving this combined editor, but we can chat!